supplementary/alica_ros1/alica_ros_turtlesim/alica_turtlesim_library/include/CircleRuntimeCondition.h

```diff
@@ -14,13 +15,28 @@ class CircleRuntimeCondition : public BasicCondition
     CircleRuntimeCondition();
     bool evaluate(std::shared_ptr<RunningPlan> rp, const IAlicaWorldModel* wm);
     // Factory method
-    static std::shared_ptr<CircleRuntimeCondition> create()
+    static std::shared_ptr<CircleRuntimeCondition> create(ConditionContext& context)
     {
+        (void) context;
         std::cerr << "Debug:"
                   << "CircleRuntimeCondition created static" << std::endl;
```

Comment:
I think we can use logging here instead. CircleRuntimeCondition::create should only be called after the logger has been initialized in AlicaContext::init

Reply:
I removed these logs, not really necessary